{"Launchpad":"https://bugs.launchpad.net/bugs/1277104","Id":"1277104","Gerrit":"https://review.openstack.org/208460","IdGerrit":"208460","IdCommit":"f353836d7fa8862c2e5fbebb2dc0e108dcc93045","Files":["cinder/tests/unit/api/v2/stubs.py","cinder/tests/unit/api/v2/test_volumes.py"],"Lines_Inseted":0,"Lines_Deleted":0,"CommitParent":"e1e33286ce2f2ed1028423691bc29b89df3da5d2","Classification":"Not_Bug","KeywordsTitle":"wrong order","Description":"wrong order, fails, shows incorrect information","KeywordsCommit":"refactors unit tests","Comments":"This error includes only test files."}